### ISSUE TITLE
waitUntilTextExists won't detect text if parent element is overflow:hidden

### ISSUE THREAD

**Issue by taboca** [electron/spectron#135]
I believe that the presence of overflow:hidden, say in the body, is preventing this infrastructure to see innerText presence using waitUntilTextExists. Is this by design and/or is this related to the Chromedriver/Webdriver parts? 

If the case is important I can land a testcase.

**Comment by MarshallOfSound** [comment]
> is this related to the Chromedriver/Webdriver parts?

Yes, almost all of the spectron API's are delivered by the `WebDriverIO` functionality
